packages/testing/src/protos/google/firebase/rules/v1/source.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package google.firebase.rules.v1;
+
+option java_package = "com.google.firebase.rules.v1";
+option java_multiple_files = true;
+option java_outer_classname = "SourceProto";
+
+// `Source` is one or more `File` messages comprising a logical set of rules.
+message Source {
+  // `File` set constituting the `Source` bundle.
+  repeated File files = 1;
```

Comment:
why would this be repeated?

Reply:
That's just the API that they came up with. I think it's forward compatibility with multi-file rulesets (once we have a real module system).